### PR TITLE
Feature/contact us

### PR DESCRIPTION
**Pull request by DeepaliAgrwl**
Contact Us story

### CODE REVIEW

**Comment by nitheshkumar-hcl** [comment]
Resolve the conflicts

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/form/button/clientlibs/js/button.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//             var valueInput = el.closest("form").find(BUTTON_VALUE);
+//             if (valueInput.val() !== "") {
+//                 if (el.val() === "") {
+//                     return el.data(PROP_ERROR_MESSAGE);
+//                 }
+//             }
+//         }
+//     });
+
+// })(jQuery);
```
Comment (nitheshkumar-hcl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/form/button/clientlibs/js/button.js]: Remove this commented block code

**Comment by nitheshkumar-hcl** [comment]
Resolve the conflicts

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/form/button/clientlibs/js/button.js (hunk 1 above)
Comment (DeepaliAgrwl) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/form/button/clientlibs/js/button.js]: removed

**Review by nitheshkumar-hcl — APPROVED** [review]
Approved the Request

**Pull request merged**